src/Axe/Models/CodeBlock.cs

```diff
@@ -17,21 +17,56 @@ public class CodeBlock
         public int Id { get; set; }
 
         /// <summary>
-        /// Gets or sets template code
+        /// Gets or sets verification code
         /// </summary>
-        [Display(Name = "Source code")]
-        [Required(ErrorMessage = "Source code is required")]
-        public string SourceCode { get; set; }
+        [Required]
+        public string VerificationCode { get; set; }
 
         /// <summary>
-        /// Gets or sets verification code
+        /// Gets or sets type of output
         /// </summary>
         [Required]
-        public string VerificationCode { get; set; }
+        public OutputTypeEnum OutputType { get; set; }
+
+        /// <summary>
+        /// Gets or sets list of test cases
+        /// </summary>
+        [Required]
+        public List<TestCaseCodeBlock> TestCases { get; set; } = new List<TestCaseCodeBlock>();
 
         /// <summary>
         /// Gets or sets task
         /// </summary>
+        [Required]
         public string Task { get; set; }
     }
+
+    /// <summary>
+    /// Enum represents type of output
+    /// </summary>
+    public enum OutputTypeEnum
+    {
```

Comment:
`enum` name doesn't need "Enum" suffix. OutputType sounds like a broad name, I suggest to use `SupportedType` name